### PR TITLE
Replaces all AI voicelines with Half Life 1 VOX

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
yeah there's like 300 less but it sounds less gay

## Changelog
:cl:
add: The AI has been granted a new and improved voice generation software. No refunds.
/:cl:

### CODE REVIEW

**Comment by Salex08** [comment]
nice

**Review by EtraIV — CHANGES_REQUESTED** [review]
how the fuck do you add a typo when the correct spelling already exists
going from `ventilation` to `ventillation`
fucking ESLs

**Comment by communist-funny-man** [comment]
Based

**Comment by karmaisblackandbluepilled** [comment]
hmm

**Comment by karmaisblackandbluepilled** [comment]
wait what

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 1]
```diff
@@ -307,7 +307,7 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/_comma.ogg',
 "inspector" = 'sound/vox_fem/inspector.ogg',
 "interchange" = 'sound/vox_fem/interchange.ogg',
 "intruder" = 'sound/vox_fem/intruder.ogg',
-"invallid" = 'sound/vox_fem/invallid.ogg',
+"invalid" = 'sound/vox_fem/invalid.ogg',
```
Comment (EtraIV) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm:310]: i thought this shit automatically figured out vox files to play automatically, wack. cool that you picked this up tho

**Review by EtraIV — APPROVED** [review]

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 1 above)
Comment (karmaisblackandbluepilled) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm:310]: is it still invallid?

Comment (EtraIV) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm:310]: kek

**Comment by EtraIV** [comment]
everyone loves this and honestly it is cool as shit. while the low word count is a problem for calling shit like suit sensors, i think it's been a great success for the testmerge

**Pull request merged**